projects/hslayers/src/components/add-data/url/add-data-url.service.ts

```diff
@@ -55,28 +55,26 @@ export class HsAddDataUrlService {
         for (const layer of service.Layer) {
           if (layer[selector] == layerToSelect) {
             layer.checked = true;
-            setTimeout(() => {
-              const id = `#hs-add-layer-${layer[selector]}`;
-              const el = this.HsLayoutService.contentWrapper.querySelector(id);
-              if (el) {
-                el.scrollIntoView();
-              }
-            }, 1000);
+            this.scrollToLayer(service[selector]);
```

Comment:
Previously you had layer[selector] not {service[selector] here. Is it ok?

Reply:
Definately no, thanks once more